srcpkgs/base-files/files/_vkpurge

```diff
@@ -0,0 +1,29 @@
+#compdef vkpurge
+
+_vkpurge() {
+	local line state
+
+	_arguments -C \
+		'1: :->subcmd' \
+		'*::kernel:->kernels'
+
+	case "$state" in
+	subcmd)
+		_values 'subcommand' \
+			'list[List removable kernel versions]' \
+			'rm[Remove removable kernel versions]'
```

Comment:
Usually the tag name is `commands` or `perfix-commands` in such cases. Looks like `_values` doesn't allow to specify tag name and always defaults to `values`. The context looks a bit messy too:

    tags in context :completion::complete:vkpurge:argument-1:
        values  (_values _vkpurge)
    tags in context :completion::complete:vkpurge:values:
        values  (_values _vkpurge)

```suggestion
		local subcommands=(
			'list:list removable kernel versions'
			'rm:remove removable kernel versions'
		)
		_describe -t commands 'subcommand' subcommands
```

`_values` calls `_describe` under the hood anyway.